test/integration/puppetlabs/puppetserver/auth_conf_test.clj

```diff
@@ -304,7 +304,10 @@
                            :sort-order 1
                            :name "all endpoints"}]}
          :webserver {:host "localhost"
-                     :port 8080}}
+                     :port 8080}
+         :puppet {"csr_attributes" (str test-resources-dir "/csr_attributes.yaml")
+                  "trusted_oid_mapping_file" (str test-resources-dir
+                                                  "/custom_trusted_oid_mapping.yaml")}}
```

Comment:
Interesting.  Hadn't thought about cases where the puppet settings were the only things missing.  In general that seems like probably an ok thing to add to the tests to work around that dependency... I think?

Do you know if there is another test that exercises these settings "for realz" so that we can validate that they are plumbed through?  Also these don't seem like the default values for these settings, so how were they getting plumbed through before?  Via puppet.conf?